core/src/impl/Kokkos_Profiling.cpp

```diff
@@ -402,6 +545,34 @@ void endDeepCopy() {}
 void initialize() {}
 void finalize() {}
 
+static EventSet current_callbacks;
```

Comment:
```
C:\projects\source\core\src\impl\Kokkos_Profiling.cpp(548,17): error C2146: syntax error: missing ';' before identifier 'current_callbacks'
```
The `MSVC` CI build doesn't enable the profiling interface and you are only including `Kokkos_Profiling.hpp` (which defines `EventSet`) if it is enabled.